### DISCUSSION TITLE
Cann't open AIdeck camera

### DISCUSSION THREAD

**Discussion by ChenMathilda** [bitcraze/discussions#606] [Q&A] [answered]
Hi,
I was doing some AIdeck example resently.
But when I was doing the test_camera experiment, I often couldn't open the camera. 
I noticed that this happens when the battery is low, but it still happens when the battery is replaced!
In addition, I used other 3 aidecks for experiments, and the camera could not be opened also. Is there any solution?
![img_v2_001e4fc2-8ce2-4137-9cb4-7f14f10a2e8g](https://user-images.githubusercontent.com/61350967/223010196-2c2520c3-8c5e-4e2e-9d0f-ba95e6ef2b52.jpg)
 
By the way, I used aideck1.1.

**Comment by hmllr** [comment, marked as answer]
Hi :)
Does this happen often or always? If only often, then I would guess the issue is in a cable/connector somewhere - in general, working with the JTAG connection is a bit fragile - if your cable has a glitchy connection somewhere it might fail just because the connection to the debugger fails. We have seen issues in connecting with the camera before, but this cases are usually rare and can be resolved with powering off, waiting around 30 seconds to be sure really all capacitors are discharged and powering on again.
I would recommend to try with the program flashed (either directly or using the bootloader and OTA update, as described in the getting started guide: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/   )
Hope this helps,
Hanna

**Reply by ChenMathilda** [reply]
> Hi :) Does this happen often or always? If only often, then I would guess the issue is in a cable/connector somewhere - in general, working with the JTAG connection is a bit fragile - if your cable has a glitchy connection somewhere it might fail just because the connection to the debugger fails. We have seen issues in connecting with the camera before, but this cases are usually rare and can be resolved with powering off, waiting around 30 seconds to be sure really all capacitors are discharged and powering on again. I would recommend to try with the program flashed (either directly or using the bootloader and OTA update, as described in the getting started guide: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/ ) Hope this helps, Hanna

Hi，
I replaced another Jtag debugging line, which do works（before replacement，I can just open camera 1 or 2 times in 10 experiments）.
But the camera still could not be opened occasionally now, I think maybe our Aideck 1.1 is too old ? It was bought two or three years ago.

**Reply by hmllr** [reply]
hmm, it shouldn't age that bad... I would still blame the JTAG (and maybe 1 in 500 tries the camera itself which can enter a strange state occasionally) and would recommend to flash the bootloader and stream images over WiFi :)
